### PR TITLE
Add Pulse v0.15.3

### PR DESCRIPTION
**Pull request by peterlewis**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/pulse.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "pulse" do
+  version "0.15.3"
+  sha256 "cfba399c46f90d11402a04137ff968ad7daf465450b5c7a3607f0f57d17904ad"
+
+  url "https://github.com/kean/Pulse/releases/download/#{version}/Pulse.app.zip"
```
Comment (vitorgalvao) [inline Casks/pulse.rb:5]: This is going to need a `verified` parameter.

File: Casks/pulse.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name "pulse"
```
Comment (vitorgalvao) [inline Casks/pulse.rb:6]:
```suggestion
  name "Pulse"
```

This is the name of the app, not a repeat of the token.

File: Casks/pulse.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  desc "Logger and Network Inspector"
+  homepage "https://kean.blog/pulse/home"
+
+  app "Pulse.app"
```
Comment (vitorgalvao) [inline Casks/pulse.rb:10]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md). See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/pulse.rb (hunk 1 above)
Comment (peterlewis) [reply Casks/pulse.rb:5]: So it will, apologies!

File: Casks/pulse.rb (hunk 3 above)
Comment (peterlewis) [reply Casks/pulse.rb:11]: Ooh that's a rather useful project! Wasn't aware of that, thank you! Zap added!

File: Casks/pulse.rb (hunk 2 above)
Comment (peterlewis) [reply Casks/pulse.rb]: So it is, apologies! Resolved!

**Review by vitorgalvao — DISMISSED** [review]

**Review by vitorgalvao — DISMISSED** [review]

**Comment by peterlewis** [comment]
Oh crumbs, thank you for picking up on the missing comma, @vitorgalvao. Much appreciated.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thanks @peterlewis!

**Pull request merged**